### PR TITLE
Enforce SSL on production-like environments

### PR DESCRIPTION
**Pull request by zspencer**
**Why?** Because we're dealing with hella sensitive data here; so HTTPS should be the de-rigor.

### CODE REVIEW

**Comment by zspencer** [comment]
[Fix: HTTPS is broken on prod with a naked URL](https://trello.com/c/lwUirm0R/201-fix-https-is-broken-on-prod-with-a-naked-url)

File: config/environments/production.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 require Rails.root.join("config/smtp")
 
 Rails.application.configure do
+  config.force_ssl = true
```
Comment (jessieay) [inline config/environments/production.rb]: should we remove the ssl-related code from `app/controllers/application_controller.rb` ?

File: config/environments/production.rb [hunk 2]
```diff
@@ -34,14 +36,4 @@
   end
 
   config.active_record.dump_schema_after_migration = false
-
-  config.paperclip_defaults = {
```
Comment (jessieay) [inline config/environments/production.rb]: this seems unrelated?

Comment (zspencer) [reply config/environments/production.rb]: It was dead code so I wanted to purge it.
